README.md

```diff
@@ -1,12 +1,13 @@
-# Connect-4
+# Connect 4
 
-### by Leo Chan, (c)1997
+### By Leo Chan, (c)1997
 
-![screenshot](screenshot.png)
+**Connect 4, otherwise sometimes known as "Four in a Row", is a classic game usually played within a plastic frame, with the objective of getting 4 pieces of the same color adjacent to each other in any direction. This program is a recreation of the game in BeOS/Haiku software.**
 
+![game in real life](irl.jpg)
 
-This is the game of Connect-4.  The game is played by players taking turns dropping pieces down a column.  The first player to get four consecutive pieces in a row, column, or diagonal wins the game.
+*This is what Connect 4 looks like in real life. Photo credit: https://www.flickr.com/photos/unloveable/2396019222*
 
-Have fun!
+![screenshot](screenshot.png)
 
-Leo 
+*This is what the game looks like in software.*
```

Comment:
instead of "in software, I'd say "in Haiku"

Reply:
solved